docs/installation/install.md

```diff
@@ -120,6 +120,8 @@ Alternatively, you can manually set a different name with the `--cluster-name` *
 ```{admonition} Note
 If you are running an [AKS private cluster](https://learn.microsoft.com/en-us/azure/aks/private-clusters), you may need to set the `--disable-api-server-sanity-check` *liqoctl* flag, since the API Server in your kubeconfig may be different from the one retrieved from the Azure APIs.
 
+If the private cluster uses private link, you can set the `--privateLink` *liqoctl* flag to use the private FQDN for the API server.
```

Comment:
Private link is same as fqdn right? What’s the difference between fqdn and private link flag? Thanks

Reply:
Hi @Sharathmk99 , the fqdn flag would be used for cases where customers setup a a custom DNS. https://learn.microsoft.com/en-us/azure/aks/private-clusters?tabs=azure-portal#configure-a-private-dns-zone

Private link with AKS is a feature where a private FQDN is generated for AKS clusters that are not accessible over the public internet. The FQDN is a generated value by Azure and stored on the AKS cluster object. In this scenario, the API server used by Liqo should be the privateFQDN, not the public FQDN. https://learn.microsoft.com/en-us/azure/private-link/private-link-service-overview

Reply:
Understood. We use private AKS cluster as well with private link and we are using fqdn to pass Private fqdn. May be there is confusion about naming conventions whether it should be ‘fqdn’ or ‘privateLink’ but use case is same. We are overwriting ApiServer with what’s present in flag.

Reply:
Thanks for responding @Sharathmk99, I understand that the existing --fqdn param can be used; but, the caller would first need to lookup the privateFQDN and pass that value in; but, that extra step is not necessary given that the value is on the AKS cluster object. Please take note of this line: https://github.com/scottboring/liqo/blob/aks-private-endpoint/pkg/liqoctl/install/aks/provider.go#L165

I considered suggesting that the existing --fqdn param be deprecated as this param is more useful; but, upon reading more it seemed like it is possible to setup a custom FQDN which I have not done with AKS and subsequently I assumed that value is not on the AKS cluster object. 

I feel this param simplifies the usage of liqoctl and is useful for my usecase. The --fqdn param would work; but requires and extra step that I would prefer to avoid.

Reply:
Makes sense. Thank you for the clarification.